pkg/defaults/policies/files/severity_high_privileged.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "b1df1abb-e5a5-4ff7-98fe-1d28a22b55d8",
-  "name": "Privileged Containers with Important and Critical Fixable CVEs",
-  "description": "Alert on containers running in privileged mode with important or critical fixable vulnerabilities",
+  "name": "Privileged containers with fixable CVEs and Severity at least Important",
```

Comment:
alerts for policy violations are raised on deployment level. hence should be worded accurately - Deployment with Privileged containers with fixable CVEs and Severity at least Important